test/jdk/javax/sound/midi/Sequencer/MetaCallback.java

```diff
@@ -101,13 +106,17 @@ static ShortMessage MidiMsg3(int a, int b, int c) {
         }
     }
     void start() {sequencer.start();}
-    void stop() {sequencer.stop();}
+    void stop() {
+        sequencer.stop();
+        sequencer.close();
+    }
```

Comment:
I can believe this was a problem.
IIRC on x64 mac I had some program block until I closed the sequencer.

Can you check other jtreg sound tests for similar potential problems

Reply:
> Can you check other jtreg sound tests for similar potential problems

Sure, i will check and if i find something i will create a separate bug listing all the cases where it might be a problem.